metadata/metadata_service/proxy/atlas_proxy.py

```diff
@@ -212,12 +250,16 @@ def _get_bookmark_entity(self, entity_uri: str, user_id: str) -> AtlasEntityWith
         except Exception as ex:
             LOGGER.exception(f'Bookmark not found. {str(ex)}')
 
-            table_entity = self._get_table_entity(table_uri=entity_uri)
+            bookmarked_entity = self._get_table_entity(table_uri=entity_uri) if resource_type == ResourceType.Table\
+                else self._get_dashboard(qualified_name=entity_uri)
```

Comment:
maybe do it explicitly if resource_type == ResourceType.Dashboard and else raise Exception